spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/DefaultTaskExecutionService.java

```diff
@@ -78,15 +86,23 @@
 
 	private static final Logger logger = LoggerFactory.getLogger(DefaultTaskExecutionService.class);
 
+	public static final String TASK_DEFINITION_DSL_TEXT = "taskDefinitionDslText";
+
+	public static final String TASK_DEPLOYMENT_PROPERTIES = "taskDeploymentProperties";
+
+	public static final String COMMAND_LINE_ARGS = "commandLineArgs";
+
+	public static final String TASK_PLATFORM_NAME = "spring.cloud.dataflow.task.platformName";
```

Comment:
There's something wrong in other parts for this name. Task gets launched with `--spring.cloud.data.flow.platformname=default` and here it's `spring.cloud.dataflow.task.platformName`.

Reply:
yeah, we need to have a commonplace to have this constant defined. Currently, they have it defined differently between the [task launcher module](https://github.com/spring-cloud/spring-cloud-dataflow/pull/3518/files#r327933282) and the [task deployment request creator](https://github.com/spring-cloud/spring-cloud-dataflow/blob/655d15ba08f05f8a9d2de18a06f4260a756d8188/spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/TaskAppDeploymentRequestCreator.java#L51)

Reply:
cc: @cppwfs

Reply:
I responded to this in PR https://github.com/spring-cloud/spring-cloud-dataflow/pull/3518